CONTRIBUTING.md

```diff
@@ -148,11 +148,26 @@ Scrub build results:
     make lint
     make dco
 
-### Save and restore dependencies
+### Restore, update and save dependencies
+
+When you make a fresh copy of the repo, all the dependecies are in `vendor/` directory for the builds to work. If you want to update the dependencies
```

Comment:
I think it reads fine in the rendered page. But, let me know if you want me to change something specific.